buzzmobile/process/gps_mapper/gps_mapper.py

```diff
@@ -12,25 +12,34 @@
 from std_msgs.msg import String
 
 #Global Variables
-bridge = CvBridge()
 frames = {}
 frames['bearing'] = frames['points'] = frames['location'] = None
 # These ranges are km dimensions of the path. We initialize them to 0.
 frames['y_range'] = frames['x_range'] = 0
 gps_model_pub = rospy.Publisher('gps_model', Image, queue_size=1)
+bridge = CvBridge()
 x_scale = y_scale = 1000 * rospy.get_param('pixels_per_m')
+line_width = int(round(rospy.get_param('pixels_per_m')
+                       * rospy.get_param('road_width')))
+sigma_x = rospy.get_param('sigma_x')
+sigma_y = rospy.get_param('sigma_y')
+image_width = rospy.get_param('image_width')
+image_height = rospy.get_param('image_height')
 
 def set_points(polyline):
     if polyline is not None:
         points = pl.decode(polyline.data)
-        y_range, x_range, top_left, bottom_right = interpolate.dimensions(frames['points'])
+        (y_range, x_range,
+         top_left, bottom_right) = interpolate.dimensions(frames['points'])
```

Comment:
Should this call to interpolate.dimensions be using `points` as arg instead of `frames['points']`? If the latter isn't set yet, then it will be `None`.